### PR TITLE
Add mountain bike network mask. Add ramp and turn channel uses.

### PR DESCRIPTION
**Pull request by dnesbitt61**

### CODE REVIEW

File: valhalla/baldr/directededge.h [hunk 1]
```diff
@@ -18,9 +18,10 @@ constexpr uint8_t kTaxiAccess       = 32;
 constexpr uint8_t kHorseAccess      = 64;
 
 // Bike Network constants. Bit constants.
-constexpr uint8_t kNcn = 1;
-constexpr uint8_t kRcn = 2;
-constexpr uint8_t kLcn = 4;
+constexpr uint8_t kNcn = 1;   // Part of national bike network
+constexpr uint8_t kRcn = 2;   // Part of regional bike network
+constexpr uint8_t kLcn = 4;   // Part of local bike network
+constexpr uint8_t kMcn = 4;   // Part of mountain bike network
```
Comment (dgearhart) [inline valhalla/baldr/directededge.h]: should this also be 4?

Comment (dgearhart) [inline valhalla/baldr/directededge.h]: seems like it should be:  kMcn = 8

**Comment by dgearhart** [comment]
:+1:

**Comment by gknisely** [comment]
:ship:

**Pull request merged**